input/intro-notes/StructureDefinition-au-core-practitioner-intro.md

```diff
@@ -32,4 +32,8 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
     </p>
 </div>
 
-<p class="request-for-feedback">Specific feedback is sought on the proposal to remove the mandatory requirement to support _revinclude of Provenance.<br/>Please comment on <a href="https://jira.hl7.org/browse/FHIR-43850">Jira issue #FHIR-43850</a>.</p>
\ No newline at end of file
+<p class="request-for-feedback">Specific feedback is sought on the proposal to remove the mandatory requirement to support _revinclude of Provenance.<br/>Please comment on <a href="https://jira.hl7.org/browse/FHIR-43850">Jira issue #FHIR-43850</a>.</p>
+
+<p class="request-for-feedback">Specific feedback is sought on the inclusion of addresses and telecom in the Practitioner's profile.<br/>Please comment on <a href="https://jira.hl7.org/browse/FHIR-43860">Jira issue #FHIR-43860</a>.</p>
+
+<p class="request-for-feedback">Following the TDG agreement to set Must Support on parts of HumanName in AU Core Practitioner, specific feedback is sought on which parts should be supported.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-44819">FHIR-44819</a>.</p>
```

Comment:
Replace
Following the TDG agreement to set Must Support on parts of HumanName in AU Core Practitioner, specific feedback is sought on which parts should be supported.

With 
Specific feedback is sought on the proposal on Practitioner.name including adding Must Support to parts of Practitioner and making Practitioner.name.family mandatory.

Reply:
Done